core/src/main/java/com/orange/ouds/core/component/OudsLink.kt

```diff
@@ -155,7 +155,7 @@ private fun OudsLink(
     val isTextOnly = icon == null && arrow == null
```

Comment:
`isTextOnly` could be renamed to `isLabelOnly`.

Reply:
I kept Figma naming which is "text only", are you sure we must change it?

Reply:
No you are right, that's a good reason for keeping it 👍